culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+reviewed.
+
+A change can take many forms, for example:
+- Adding a new guide or piece of documentation
+- Adding a new module to the codebase
+- Modifying an existing guide or module
+
+
+## Participants in the Review Process
+
+### Contributor
+
+The contributor is the person proposing a change. Anyone can decide to be a
+contributor: the only requirements are that you have a change you need to
+propose, and that you agree to be a respectful participant in the review
+process. (When we have a code of conduct--link to it here.)
+
+### Reviewer
+
+The reviewers are the people who review the change, suggest changes to it, and
+eventually approve it. Anyone in the community can act as a reviewer: the only
+requirements are that you have some insight to share (or questions to ask)
+about the change, and that you agree to be a respectful participant in the
+process.
+
+### Maintainer
+
+Maintainers are responsible for the health and maintenance of a portion of the
+project. They are the stewards of the review process, and have the privileges
+needed to move forward or block a change. Not everyone can be a maintainer; it
+requires deep knowledge of the project, high trust from the community,
+experience as both a contributor and reviewer, and substantial commitment.
+Since we invest maintainers with special privileges and trust, we have high
+expectations of them. Those acting as maintainers must be respectful,
+considerate, and judicious in the way they exercise their powers.
+
+## Lifecycle of a Review
+
+### Proposal
+The first step is for a contributor to propose a _change_.
+The contributor needs to do two things at this stage:
+- Flesh out the change (e.g. writing the code), so that there is something to review
+- Explain the change (e.g. writing a pull request description), so that reviewers can easily understand it
+
+Both steps are vital, and contributors should put real effort into explaining the change well.
+Doing so is respectful of reviewers' time, and makes it easier for others in the community to follow
+along with the project and get involved themselves.
+
+Finally, before requesting a review, the contributor should _self-review_ their change.
+It's amazing how many issues can be spotted and fixed during self-review.
+This is respectful of reviewers' time, and makes the process a lot smoother.
+
+### Review
+
+Once a change is ready and has been self-reviewed, it's ready for review. Since
+most SourceCred artifacts are tracked via repositories on GitHub, this will
+usually mean creating a pull request.
+
+Once the change is proposed, anyone in the project can add their own review.
+The person proposing the change can also request specific reviewers, if certain
+individiuals have context or knowledge thet will make their reivew particularly
+valuable.
+
+On finishing their review, the reviewer can:
+- Ask questions to better understand the change
+- Suggest changes to it
+- Approve the change
+
```

Comment:
I guess we don't want to consider the reality that some changes don't get accepted, and get a wont-fix?

Reply:
Good point. Added a line mentioning this case.